shared/js/ui/base/chrome-ui-wrapper.es6.js

```diff
@@ -28,8 +28,8 @@ let getBackgroundTabData = () => {
     })
 }
 
-let createBrowserTab = (url) => {
-    window.chrome.tabs.create({url: `${url}&bext=${window.localStorage['os']}cr`})
+let search = (url) => {
+    window.chrome.tabs.create({url: `https://duckduckgo.com/?q=${url}&bext=${window.localStorage['os']}cr`})
 }
```

Comment:
Tweaked this function because the name was no longer an accurate representation of what it was actually used for (even before this PR, we wouldn't want to add the `bext` parameter on *any* browser tab opened by the extension).

It's not used elsewhere.